Connected web3 provider at 10.195.2.242:8008
Date.now(): 1579018729215 startTime: 1579018834272 delay: 105061
Deploying smart contract from 0x6fff54aa0710e9b47f5999f129f3ed301ea58d5a
TX HASH:
 0x83b22038aae97f0d4536837b10053f39c83aab1fd218642cc4f8d209633e5619
RECEIPT:
 {
  blockHash: '0x9418edc8a11d1401cf3946d0cbaff7d340b958392e0f9e19c9c6c808bd46ff9a',
  blockNumber: 100,
  contractAddress: '0x8f6983a2F40df2F1d456A440a6f003AF231B7ef7',
  cumulativeGasUsed: 218495,
  from: '0x6fff54aa0710e9b47f5999f129f3ed301ea58d5a',
  gasUsed: 218495,
  logs: [],
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  status: true,
  to: null,
  transactionHash: '0x83b22038aae97f0d4536837b10053f39c83aab1fd218642cc4f8d209633e5619',
  transactionIndex: 0
}
Contract at: 0x8f6983a2F40df2F1d456A440a6f003AF231B7ef7

Evaluating  6400txs...

Sending TXS started at 1579018834361 ...
startingBlock: 205
(node:20087) UnhandledPromiseRejectionWarning: Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
(node:20087) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:20087) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
ERROR:
 Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
========================================================
========================================================

/home/ubuntu/benchprezz/benchmark/tpsUnsigned.js:113
                throw 'Blockchain capabilities EXCEEDED - ABORTING test run!';
                ^
Blockchain capabilities EXCEEDED - ABORTING test run!
